pkg/didcomm/transport/http/outbound_test.go

```diff
@@ -0,0 +1,183 @@
+/*
+	Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+	SPDX-License-Identifier: Apache-2.0
+*/
+
+package http
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+const certPrefix = "../../../../test/fixtures/keys/"
```

Comment:
this will need to be updated in the next PR.

Reply:
yes, next PR has the go generate command with updated path..